catalog/core/catalog-core-localstorageprovider/src/test/java/org/codice/ddf/catalog/content/impl/FileSystemStorageProviderTest.java

```diff
@@ -122,7 +132,8 @@ public void setUp() throws IOException {
 
   @After
   public void clean() {
-    tempFolder.delete();
+    System.clearProperty(SecurityConstants.KEYSET_DIR);
```

Comment:
❓ Do we not need that deletion?

Reply:
No, the temp folder is automatically created/deleted per test method when the `@Rule` annotation is specified at it's initialization:
```
@Rule public TemporaryFolder temporaryFolder = new TemporaryFolder();
```